make/test.mk

```diff
@@ -75,6 +75,9 @@ test-olm-integration: push-operator-image olm-integration-setup
 	$(Q)sed -e "s,REPLACE_IMAGE,$(DEVCONSOLE_OPERATOR_REGISTRY_IMAGE):$(devconsole_version)-$(TAG)," ./test/e2e/catalog_source.yaml | oc apply -f -
 	$(Q)oc apply -f ./test/e2e/subscription.yaml
 
+	# The following cleanup is required due to a potential bug in the test framework.
```

Comment:
The clean up should go under `olm-integration-cleanup` target. I missed it earlier as I reviewed this from my mobile ;-)

Reply:
That target is called at the end or start of the test right?

Reply:
start of the test.